Dockerfile

```diff
@@ -108,6 +108,12 @@ EXPOSE 3000
 ENTRYPOINT [ "bin/docker-entrypoint" ]
 CMD [ "rails", "server" ]
 
+# ------------------------------------------------------------------------------
+# shellcheck
+# ------------------------------------------------------------------------------
+
+FROM koalaman/shellcheck:stable AS shellcheck
```

Comment:
What's the advantage in using the Docker image, rather than doing `apt-get install shellcheck`?

Reply:
We're using Alpine, which doesn't have ShellCheck in its repositories.

Reply:
Aaaaah, I did wonder if it was Alpine-related